lightning/src/ln/channel.rs

```diff
@@ -288,7 +288,7 @@ impl HTLCCandidate {
 }
 
 /// Information needed for constructing an invoice route hint for this channel.
-#[derive(Clone)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Instead of implementing `PartialEq`, test that `fee_base_msat` hasn't changed.

Reply:
I think its generally a good thing to have either way - its a public struct and its reasonable for users to want to compare it?